README.md

```diff
@@ -29,57 +29,68 @@ const Pet = require('../src/pet');
 
 ## Let's have the pet now and take care ot it
 **Create and name your pet**
+Codes to be copied to node are in this style
```

Comment:
Probably the most complete readme I've read in this project so far.. Good job on that!